biz_portal/apps/portal/templates/portal/business_detail.html

```diff
@@ -5,6 +5,38 @@
   <div class="business-details-section">
     <div class="layout">
       <div>
+        <div class="business-menu">
+            <button class="mdc-button business-menu-button" id="show">
+              <i class="material-icons mdc-button__icon" aria-hidden="true">more_horiz</i>
+            </button>
+        </div>
+        <div class="mdc-dialog"
+           role="alertdialog"
+            aria-modal="true"
+            aria-labelledby="my-dialog-title"
+            aria-describedby="my-dialog-content">
+          <div class="mdc-dialog__container">
+            <div class="mdc-dialog__surface">
+              <div class="mdc-dialog__content" id="my-dialog-content">
+                <ul class="mdc-list mdc-list--avatar-list">
+                  <li class="mdc-list-item" tabindex="0" data-mdc-dialog-action="none">
+                    <span class="mdc-list-item__text">Share</span>
+                  </li>
+                  <li class="mdc-list-item" data-mdc-dialog-action="callisto">
```

Comment:
Not sure if callisto required. I think it was just an example. Remove

Reply:
Done